webapps/frontend/app/viewmodels/wizards/addpreset/data.js

```diff
@@ -28,13 +28,22 @@ define(['durandal/app', 'knockout', 'jquery'], function(app, ko, $){
             encryptionOptions:  ko.observableArray(['aes-cbc-256', 'none']),
             encryption:         ko.observable('none'),
             policies:           ko.observableArray([]),
-            currentPresets:     ko.observableArray([])
+            currentPresets:     ko.observableArray([]),
+            currentPreset:      ko.observable(),
+            editPreset:         ko.observable(false),
+            canEdit:            ko.observable(true)
         };
+
+        data.canEdit = ko.computed(function() {
+            return !data.editPreset()
+        });
+
         data.cleanPolicies = ko.computed(function() {
             var policies = [], i = 0, replication = data.replication() - 1;
             if (data.advanced()) {
                 $.each(data.policies(), function (index, policy) {
-                    policies.push([policy.k(), policy.m(), policy.c(), policy.x()]);
+                    policies.push([parseInt(policy.k()), parseInt(policy.m()), parseInt(policy.c()),
```

Comment:
This parseInt might result in unwanted behavior depending on what the customers enter (undefined, null, ...)